_variables.tf

```diff
@@ -71,8 +71,15 @@ variable "mod_dependency" {
   description = "Dependence variable binds all AWS resources allocated by this module, dependent modules reference this variable."
 }
 
+variable "iam_policy_attach" {
+  type        = string
```

Comment:
@7adityaraj please change the variable type to boolean.

Reply:
@caiovfernandes my bad, thanks for reviewing